installer/conf/omsagent.d/baseline.conf

```diff
@@ -0,0 +1,30 @@
+# Baseline plugins
+
+<source> 
+    type exec
+    tag oms.baseline_immediate_run
+    command /opt/microsoft/omsagent/plugin/omsbaseline -d /etc/opt/microsoft/omsagent/conf/omsagent.d 

```

Comment:
Why only omsbaseline file? What are about all other files, for example oms_audits.xml? How following files will be in bundle?